imageroot/actions/get-route/validate-output.json

```
@@ -123,6 +126,11 @@
             "title": "Strip prefix path",
             "description": "Strip the path prefix from the request"
         },
+        "skipCertVerify": {
```

```suggestion
        "skip_cert_verify": {
```
I prefer the use of the snake case instead of the camel case, to remain consistent with the style of the other fields.

---

done